docs/roman/datamodels/metadata.rst

```diff
@@ -8,74 +8,44 @@ Metadata information associated with a data model is accessed through
 its `meta` member.  For example, to access the date that an
 observation was made::
 
-    print(model.meta.observation.date)
+    print(model.meta.observation.start_time)
 
 Metadata values are automatically type-checked against the schema when
 they are set. Therefore, setting a attribute which expects a number to a
-string will raise an exception::
-
-.. doctest-skip::
-
-    >>> from romancal.datamodels import ImageModel
-    >>> model = ImageModel()
-    >>> model.meta.target.ra = "foo"    # doctest: +SKIP
-    Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-      File "site-packages/romancal.datamodels/schema.py", line 672, in __setattr__
-        object.__setattr__(self, attr, val)
-      File "site-packages/romancal.datamodels/schema.py", line 490, in __set__
-        val = self.to_basic_type(val)
-      File "site-packages/romancal.datamodels/schema.py", line 422, in to_basic_type
-        raise ValueError(e.message)
-    ValueError: 'foo' is not of type u'number'
+string will raise an exception.
+
+.. code-block:: python
+
+        from romancal.datamodels import ImageModel
```

Comment:
Doesn't romancal.datamodels not exist anymore?

```python
In [1]: from romancal.datamodels import ImageModel
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-284b0ab95ba6> in <module>
----> 1 from romancal.datamodels import ImageModel

ImportError: cannot import name 'ImageModel' from 'romancal.datamodels' (unknown location)
```

Reply:
@ddavis-stsci You must be working off an old code base. Yo may want to do `git clean -fxd` in your working directory. Note that this command will remove everything that is not in the repo.

Reply:
Updated this. It was a holdover.